Command/CleanMediaCommand.php

```diff
@@ -121,14 +96,11 @@ private function mediaExists($filename, $context = null)
 
         if (count($fileParts) > 1 && $fileParts[0] == 'thumb') {
             return $mediaManager->findOneBy(array(
-                    'id' => $fileParts[1],
-                    'context' => $context,
-                )) != null;
+                'id' => $fileParts[1],
+                'context' => $context,
+            )) != null;
         }
 
-        return count($mediaManager->findBy(array(
-                'providerReference' => $filename,
-                'providers' => $this->getProviders(),
```

Comment:
This line was causing the issue, the rest of the changes on this PR are just a refactor of removing this line:

- `$this->getProviders()` is no longer used
- verbosity was not handled correctly
- $providers private parameter is no longer needed